app-admin/paxtest/paxtest-0.9.15-r2.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="PaX regression test suite"
+HOMEPAGE="https://pax.grsecurity.net"
+SRC_URI="https://grsecurity.net/~spender/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86"
+
+DEPEND="sys-apps/paxctl"
```

Comment:
Is this really DEPEND only? It might be, but it looks odd, dunno

Reply:
paxctl seems like only installs an executable. so I'm pretty sure it should be in BDEPEND...

But I'll leave it there until someone from the hardened team gives me the ok.

Reply:
Does paxtest call it at runtime or something?